### PR TITLE
Eula full screen

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 4138 .

#### Changes proposed in this Pull Request:

- Eula Full screen component
- Added screen inside Registration workflow
- Rename RegistrationUIActions => RegistrationUIActionsLegacy for example1 

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2023-06-08 at 9 05 26 PM

#### To Test:

- http://localhost:3000/registration-workflow
- Also please test Example 1

#### Any specific feedback you are looking for?
-

### CODE REVIEW

File: login-workflow/src/auth-shared/index.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export * from './contexts/SecurityContextProvider';
 export { AuthUIContextProvider } from './contexts/AuthUIContextProvider';
 export type {
     AuthUIContextProviderProps,
-    RegistrationUIActions,
+    // RegistrationUIActions,
```
Comment (ektaghag-eaton) [inline login-workflow/src/auth-shared/index.ts:23]: I have removed RegistrationUIActions from here. that's why the build is failing for example 1. Do I need to update the Build file? @JeffGreiner-eaton , @daileytj

Comment (JeffGreiner-eaton) [reply login-workflow/src/auth-shared/index.ts:23]: Is this the error you see ?

image

Comment (ektaghag-eaton) [reply login-workflow/src/auth-shared/index.ts]: Rename  RegistrationUIActions  => RegistrationUIActionsLegacy for example1 https://github.com/etn-ccis/blui-react-workflows/pull/302/commits/fcb5f2dae9044b003f0080af790679182acc4c9a. Build is not failing now. @JeffGreiner-eaton please review it. Thanks!!!

**Comment by manojleaton** [comment]
Eula full screen in Registration workflow is looks good to me

File: login-workflow/src/new-architecture/screens/EulaScreen/EulaScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        void loadAndCacheEula();
+    }, [loadAndCacheEula]);
+
+    return (
+        <EulaScreenBase
+            WorkflowCardHeaderProps={{ title: title }}
+            eulaContent={eulaData}
+            WorkflowCardBaseProps={{
+                loading: eulaLoaded,
+            }}
```
[the remark this answers is not in the document]
Comment (ektaghag-eaton) [reply login-workflow/src/new-architecture/screens/EulaScreen/EulaScreen.tsx:84]: I followed this description for using `loading` prop of WorkflowCardBaseProps.
Screenshot 2023-06-14 at 10 46 04 PM
BLUI-4135. If we want to add a new prop in `EulaScreenaBase` I will raise an issue and fix it.

[the remark this answers is not in the document]
Comment (JeffGreiner-eaton) [reply login-workflow/src/new-architecture/screens/EulaScreen/EulaScreen.tsx:84]: we can add this to the backlog to add a loading prop to the Eula screen base but will be low priority for now and will update later

**Review by JeffGreiner-eaton — APPROVED** [review]

**Pull request merged**